packages/nimble/inst/include/nimble/EigenTypedefs.h

```diff
@@ -85,22 +85,72 @@ nimSmartPtr<EIGEN_SVDCLASS_R>   EIGEN_SVD_R(const Eigen::MatrixBase<Derived> &x,
 }
 
 template<class Derived>
-void EIGEN_EIGEN_INTERNAL(const Eigen::MatrixBase<Derived> &x, bool valuesOnly, EIGEN_EIGENCLASS_R *returnClass) {
-  // nimSmartPtr<EIGEN_EIGENCLASS> returnClass = new EIGEN_EIGENCLASS;
-  returnClass->getValues().initialize(0, 0, x.rows());
+bool EIGEN_CHECKSYMMETRY(const Eigen::MatrixBase<Derived> &x) {
```

Comment:
nit: In C++ a common convention is to reserve ALL_CAPS_SNAKE_CASE identifiers for macros. I would recommend renaming to something like `is_symmetric`.